notebooks/MINDEvalResults.md

```diff
@@ -0,0 +1,140 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Offline Evaluation Metrics Visualizations
+This notebook visualizes user-specific performance metrics of various recommenders in the MIND validation dataset to assess effectiveness and ranking overlap. We explore two metric groups:
+1. **Effectiveness Metrics**: We use ranking-based metrics, Normalized Discounted Cumulative Gain (NDCG) and Mean Reciprocal Rank (MRR), to evaluate recommender effectiveness.
+2. **Ranking Overlap Metrics**: We use Rank-Based Overlap (RBO) to assess consistency in top-k recommendations relative to fianl rankings.
+
++++
+
+## 1. Setup
+
+```{code-cell} ipython3
+# Importing Libraries
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from scipy.stats import sem, t
+from IPython.display import display, HTML
+
+# Loading Data
+mind_val_user_metrics = pd.read_csv('../outputs/mind-val-user-metrics.csv.gz')
+mind_val_user_metrics.head()
+```
+
+## 2. Visualizations
+To evaluate the plot, look for higher values in the bars. Higher values indicate better performance in terms of the respective metric. Error bars show the variability of the results. The exact values of the error bars are provided in the summary table below.
```

Comment:
This is not always true — RBO isn't “better”, it's just documenting a change. For our effectiveness metrics, higher is better.

Let's put the interpretation notes with each metric family.